test/unit/auth/authenticate.test.js

```diff
@@ -64,7 +64,7 @@ const { authDirectiveTransformer, checkAuthGroup, getAuth, authGroups } = await
   '../../../app/auth/authenticate.js'
 )
 
-describe('getJwtPublicKey', () => {
+describe.skip('getJwtPublicKey', () => {
```

Comment:
Presumably this test is still needed?
```suggestion
describe('getJwtPublicKey', () => {
```